src/fiber/serde_utils.rs

```diff
@@ -99,7 +99,7 @@ macro_rules! uint_as_hex {
             pub $name,
             $ty,
             |u: &$ty| format!("0x{:x}", u),
-            |hex: &str| -> Result<$ty, String> {
+            |hex: String| -> Result<$ty, String> {
```

Comment:
why change this type?

Reply:
An error occured when deserialize U128Hex with `toml`.

``` txt
[2025-02-18T07:42:59Z INFO  fiber_autopilot] Using config file: fiber-autopilot.toml
Error: TOML parse error at line 5, column 19
  |
5 | available_funds = "0xE8D4A51000"
  |                   ^^^^^^^^^^^^^^
invalid type: string "0xE8D4A51000", expected a borrowed string
```

This line [fiber-autopilot](https://github.com/jjyr/fiber-autopilot/blob/master/src/config.rs#L45) caused the error.


I have not fully understand this issue, but follow the suggestion can fix this error https://github.com/jonasbb/serde_with/issues/620